export const MockedExceptions = {
    NODE_WITH_SPACES: {
        name: 'Error',
        message: '',
        stack: 'Error\n'
        +'    at Object.<anonymous> (/var/app/scratch/my project/index.js:2:9)\n'
        +'    at Module._compile (internal/modules/cjs/loader.js:774:30)\n'
        +'    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)\n'
        +'    at Module.load (internal/modules/cjs/loader.js:641:32)\n'
        +'    at Function.Module._load (internal/modules/cjs/loader.js:556:12)\n'
        +'    at Function.Module.runMain (internal/modules/cjs/loader.js:837:10)\n'
        +'    at internal/main/run_main_module.js:17:11'
    }
};